app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class='container my-5'>
+  <div class='row'>
+    <div class='col-12'>
+
+      <%= link_to [:new, :admin, :user], class: 'btn btn-primary float-right mb-3' do %>
+        <%= icon('plus') %>
+        <%= I18n.t('users.add') %>
+      <% end %>
+
+      <div class='table-responsive'>
+        <table class='table table-bordered'>
+          <thead>
+            <tr>
+              <th>
+                <%= I18n.t('user.name') %>
+              </th>
+              <th>
+                <%= I18n.t('user.email') %>
+              </th>
+              <th style="width: 70px;">
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <% @users.each do |user| %>
+              <% cache user do %>
+                <tr>
+                  <td>
+                    <%= user.name %>
+                    <% if user.admin? %>
+                      <br>
+                      Admin
+                    <% end %>
+                  </td>
+                  <td>
+                    <%= user.email %>
+                  </td>
+                  <td>
+                    <%= link_to [:edit, :admin, user] do %>
+                      <%= icon('pencil') %>
+                    <% end %>
+                    <%= link_to [:admin, user], method: :delete, data: { confirm: I18n.t('users.confirm') } do %>
+                      <%= icon('trash') %>
+                    <% end %>
+                  </td>
```

Comment:
Currently if you hover over the edit button you get this underlined link CSS:

<img width="148" alt="Screenshot 2020-09-24 at 18 15 45" src="https://user-images.githubusercontent.com/7175262/94179859-0d016980-fe95-11ea-8c16-c276d7a26748.png">

One solution is to use the bootstrap buttons instead, but kind of needs to be in separate columns to work:

```suggestion
                  <td>
                    <%= link_to [:edit, :admin, user], class: 'btn btn-link' do %>
                      <%= icon('pencil') %>
                    <% end %>
                  </td>
                  <td>
                    <%= link_to [:admin, user], method: :delete, data: { confirm: I18n.t('users.confirm') }, class: 'btn btn-link'  do %>
                      <%= icon('trash') %>
                    <% end %>
                  </td>
```

Looks like this:

<img width="847" alt="Screenshot 2020-09-24 at 18 38 53" src="https://user-images.githubusercontent.com/7175262/94179989-3cb07180-fe95-11ea-9603-dbf19e469b7e.png">

Reply:
Nice spot